src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -1365,15 +1382,15 @@ private XmlWriter CreateWriterForInnerOuterXml(StringWriter sw)
 
         private void SetNamespacesFlag(XmlTextWriter xtw)
         {
-            XmlTextReader tr = this as XmlTextReader;
+            XmlTextReader? tr = this as XmlTextReader;
```

Comment:
NIT: We normally update it to use `s` instead

Reply:
what do you mean?

Reply:
oh sorry `i` from `is` didn't make it 😛, fixed 
```suggestion
            if (this is XmlTextReader tr)
```